linuxNew/jdk/debian/src/main/packaging/temurin/8/debian/changelog

```diff
@@ -0,0 +1,5 @@
+temurin-8-jdk (8.0.302.0.0+8-0~adopt0) UNRELEASED; urgency=medium
+
+  * Eclipse Temurin 8.0.302.0.0+8-0~adopt0 release.
+
+ -- Eclipse Adoptium Package Maintainers <packaging@adoptium.com>  Fri, 22 Jan 2021 14:32:55 +0000
```

Comment:
As suggested in #331 we should perhaps use an email alias that actually exists. `temurin-dev@eclipse.org` was the suggestion.